applications/GeoMechanicsApplication/tests/cpp_tests/test_generalized_newmark_T_scheme.cpp

```diff
@@ -0,0 +1,180 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_strategies/schemes/generalized_newmark_T_scheme.hpp"
+#include "spaces/ublas_space.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+using SparseSpaceType = UblasSpace<double, CompressedMatrix, Vector>;
+using LocalSpaceType = UblasSpace<double, Matrix, Vector>;
+
+namespace Kratos::Testing {
+
+GeneralizedNewmarkTScheme<SparseSpaceType, LocalSpaceType> CreateValidScheme()
+{
+    GeneralizedNewmarkTScheme<SparseSpaceType, LocalSpaceType> result(0.75);
+    return result;
```

Comment:
I would merge these two lines:
```suggestion
    return GeneralizedNewmarkTScheme<SparseSpaceType, LocalSpaceType> result(0.75);
```

Reply:
Done